Sources_v3/StreamChatUI/Common Views/OnlineIndicatorView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  OnlineIndicatorView.swift
+//  StreamChatUI
+//
+//  Created by Dominik Bucher on 25/11/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+class OnlineIndicatorView: UIView {
+    public var fillColor: UIColor = .green {
```

Comment:
nit: this can be `systemGreen`

Reply:
Not nit :D You are actually right, however is ugly with the check for #availability.  Anyway I created static color property which switches this.

Reply:
My bad, `.systemGreen` is since iOS 7, changed it :)

Reply:
Yeah, iOS 13 `.systemBackground` brings confusion here 😄